examples/tensorflow/yolo_v5_real_time_with_labels.py

```diff
@@ -0,0 +1,152 @@
+# $ pip3 install tflite-runtime
+# $ pip3 install opencv-python==4.4.0.46
+# $ pip3 install pillow
+# $ pip3 install numpy
+#
+# and run from the command line,
+#
+# python3 yolo_v5_real_time_with_labels.py --model=yolov5s-fp16.tflite --label=coco_labels_yolov5.txt
+
+import argparse
+
+import cv2
+import numpy as np
+import tflite_runtime.interpreter as tflite
+
+from picamera2 import MappedArray, Picamera2, Preview
+
+normalSize = [1920,1080]
+lowresSize = [1280, 720]
+
+rectangles = []
+
+
+def ReadLabelFile(file_path):
+    with open(file_path, 'r') as f:
+        lines = f.readlines()
+    ret = {}
+    for line in lines:
+        pair = line.strip().split(maxsplit=1)
+        ret[int(pair[0])] = pair[1].strip()
+    return ret
+
+
+def DrawRectangles(request):
+    with MappedArray(request, "main") as m:
+        for rect in rectangles:
+            xmin, ymin, xmax, ymax = int(rect[0]), int(rect[1]), int(rect[2]), int(rect[3])
+
+            rect_start = xmin,ymin
+            rect_end = xmax,ymax
+            cv2.rectangle(m.array, rect_start, rect_end, (0, 255, 0, 0))
+            if len(rect) == 5:
+                text = rect[4]
+                font = cv2.FONT_HERSHEY_SIMPLEX
+                cv2.putText(m.array, text, (xmin,ymin-10),
+                            font, 1, (255, 255, 255), 2, cv2.LINE_AA)
+        
+def classFilter(classdata):
+    classes = []  # create a list
+    for i in range(classdata.shape[0]):         # loop through all predictions
+        classes.append(classdata[i].argmax())   # get the best classification location
+    return classes  # return classes (int)
+
+def YOLOdetect(output_data):  # input = interpreter, output is boxes(xyxy), classes, scores
+    output_data = output_data[0]                # x(1, 25200, 7) to x(25200, 7)
+    boxes = np.squeeze(output_data[..., :4])    # boxes  [25200, 4]
+    scores = np.squeeze( output_data[..., 4:5]) # confidences  [25200, 1]
+    classes = classFilter(output_data[..., 5:]) # get classes
+    # Convert nx4 boxes from [x, y, w, h] to [x1, y1, x2, y2] where xy1=top-left, xy2=bottom-right
+    x, y, w, h = boxes[..., 0], boxes[..., 1], boxes[..., 2], boxes[..., 3] #xywh
+    xyxy = [x - w / 2, y - h / 2, x + w / 2, y + h / 2]  # xywh to xyxy   
+
+    return xyxy, classes, scores  # output is boxes(x,y,x,y), classes(int), scores(float) [predictions length]
+
+
+def main():
+    global rectangles
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--model', help='Path of the detection model.', required=True)
+    parser.add_argument('--label', help='Path of the labels file.')
+    args = parser.parse_args()
+    
+    if args.label: 
+        labels = ReadLabelFile(args.label)
+    else:
+        labels = None
+    picam2 = Picamera2()
+    picam2.start_preview(Preview.QTGL)
+    
+    stream_format = "YUV420"
+    if Picamera2.platform == "Platform.PISP":
```

Comment:
Does this work? I think I would have expected
```
    if Picamera2.platform == Platform.PISP:
```
(and import `Platform` from `picamera2` at the top).

Reply:
I was programming late, i checked its type and it is indeed enum and not string, i fixed this in the new commit